test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -317,6 +317,34 @@ export function setup(opts: minimist.ParsedArgs) {
 				await app.code.dispatchKeybinding('escape');
 				await app.workbench.sqlNotebook.waitForTextCellPreviewContent(sampleLabel, `p a[href="${sampleAddress}"]`);
 			});
+
+			it('can use keyboard shortcuts for cell toolbar actions', async function () {
+				const app = this.app as Application;
+				await app.workbench.sqlNotebook.newUntitledNotebook();
+				await app.workbench.sqlNotebook.addCellFromPlaceholder('Markdown');
+				await app.workbench.sqlNotebook.waitForPlaceholderGone();
+				await app.workbench.sqlNotebook.textCellToolbar.changeTextCellView('Markdown View');
+				let testText = 'Markdown Keyboard Shortcut Test';
+				await app.workbench.sqlNotebook.waitForTypeInEditor(testText);
+				await app.workbench.sqlNotebook.selectAllTextInEditor();
+				const ctrlOrCmd = process.platform === 'darwin' ? 'cmd' : 'ctrl';
```

Comment:
We already have this in https://github.com/microsoft/azuredatastudio/blob/40a2181953a64566730f7827db7490ecd113d216/test/automation/src/sql/notebook.ts